Services/sdLitica.Services.Test/sdLitica.Services.Test.csproj

```diff
@@ -1,16 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Yes, we should create all libraries, i.e. not runnable project, using netstandard. I think 2.0 is the current most recent version. What do you think? 
Test project needs to be a runnable one. 2.1 is the same as WebAPi target.

Also, there is a recent 2.2 SDK. Maybe we should migrate ASAP. What do you think?

Reply:
Let's try targeting NetCore 2.2 everywhere since 2.3 is not expected and will be most recent for several months. Microsoft is investing in 3.0 already, but it will not be released soon, only early beta available.

Reply:
OK! So, we should migrate WebApi project and Test project to 2.2.
Dont you agree that DLL should be target NetStandard?